ewwep.py

```diff
@@ -930,6 +930,13 @@ def explode(damage = 0, district_data = None):
 			value = value.lower()
 
 		weapon = ewcfg.weapon_map.get(value)
+
+		if weapon != None:
+			if weapon.id_weapon == 'pickaxe':
+				weapon = None
+			else:
+				weapon = weapon
```

Comment:
this is terrible. we should give weapons some sort of source tag, like item vendors, and check that here